public/src/overmind/actions.tsx

```diff
@@ -547,44 +587,44 @@ export const createOrUpdateCriterion = async ({ effects }: Context, { criterion,
     }
 
     // Existing criteria have a criteria id > 0, new criteria have a criteria id of 0
-    if (criterion.ID && success(await effects.grpcMan.updateCriterion(criterion))) {
+    if (criterion.ID && success(await effects.api.client.updateCriterion(criterion))) {
```

Comment:
We probably want to follow the pattern also here and move the call into the if statement and do:
```ts
    if (response.error) {
        return
    }
```